services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -69,15 +83,161 @@ const Asset = ({ libraryData }) => {
+      )
+    }
+
+    return (
+      <ul className={styles.meta}>
+        {properties.map((prop, i) => (
+          <li key={i}>{prop === 'status' && renderStatus()}</li>
+        ))}
+      </ul>
+    )
+  }
+
   return (
     <>
       <PageHeader title={seo.title} pictogram={TextInput} />
       <PageBreadcrumb items={breadcrumbItems} />
+      <Dashboard className={styles.content}>
+        <Column className={dashboardStyles.column}>
+          <DashboardItem
+            aspectRatio={{ sm: '2x1', md: '1x1', lg: '3x4', xlg: '1x1' }}
+            border={['sm']}
+          >
+            <p className={dashboardStyles.title}>Library</p>
+            <h3 className={dashboardStyles.titleLarge}>{libraryData.content.name}</h3>
+            <Link href={`${assetPath}`}>
+              <a className={clsx(styles.metaLinkLarge, dashboardStyles.subcolumn)}>
+                {'v.' + libraryData.content.version}
+              </a>
+            </Link>
+            {SponsorIcon && <SponsorIcon className={styles.metaAbsolute} size={32} />}
+          </DashboardItem>
+        </Column>
+        <Column className={dashboardStyles.column} lg={2}>
+          <DashboardItem aspectRatio={{ sm: '1x1', lg: 'none', xlg: 'none' }} border={['sm']}>
+            <Grid columns={2} className={dashboardStyles.subgrid}>
+              <Column className={clsx(styles.metaInfo, dashboardStyles.subcolumn)}>
+                <p className={dashboardStyles.title}>Sponsor</p>
+                <Link href={`${assetPath}`}>
+                  <a className={styles.metaLink}>{libraryData.content.name}</a>
+                </Link>
+              </Column>
+              <Column>
+                <p className={clsx(dashboardStyles.title, dashboardStyles.subcolumn)}>Type</p>
+                <h3 className={dashboardStyles.subtitle}>{assetType}</h3>
+              </Column>
+              <Column>
+                <p className={clsx(dashboardStyles.title, dashboardStyles.subcolumn)}>Framework</p>
+                <h3 className={dashboardStyles.subtitle}>{framework}</h3>
+              </Column>
+              <Column>
+                <p className={clsx(dashboardStyles.title, dashboardStyles.subcolumn)}>Status</p>
+                <AssetItemMeta
+                  className={clsx(dashboardStyles.title, styles.meta)}
+                  properties={['status']}
+                />
+              </Column>
+              <Column className={dashboardStyles.subcolumnLinks}>
+                <p className={clsx(dashboardStyles.title)}>Demo links</p>
+                <Link href={`${assetPath}`}>
+                  <a className={styles.metaLink}>{libraryData.content.name}</a>
+                </Link>
+                <Link href={`${assetPath}`}>
+                  <a className={styles.metaLink}>{libraryData.content.name}</a>
+                </Link>
+              </Column>
+              <Column>
+                <p className={clsx(dashboardStyles.title, dashboardStyles.subcolumn)}>Type</p>
```

Comment:
Should this be "Tag"?

Reply:
Hey Matt, just wanted to clarify, what you meant by `Tag` here? Did you mean it needs to be styled like our Carbon Tag? Just wanted to clarify 

![Uploading Screen Shot 2022-02-14 at 10.47.55 PM.png…]()

Reply:
@andreancardona bottom of that screenshot, it should say "Tag" not "Type" twice.

Reply:
OMG I see what you mean haha sorry about that, good catch